src/server/db_slice.cc

```diff
@@ -1049,18 +1051,15 @@ DbSlice::ItAndExp DbSlice::ExpireIfNeeded(const Context& cntx, PrimeIterator it)
     return {it, expire_it};
 
   string tmp_key_buf;
-  string_view tmp_key;
+  string_view tmp_key = it->first.GetSlice(&tmp_key_buf);
```

Comment:
please revert the changes here to be as before. you did not changed this function

Reply:
done